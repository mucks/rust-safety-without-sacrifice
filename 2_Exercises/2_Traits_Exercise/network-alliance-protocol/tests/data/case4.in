# Trait objects summary
STATE Zuzalu 400
CITIZEN Zuzalu 1 Zoe Builder
CITIZEN Zuzalu 2 Zack Researcher
CITIZEN Zuzalu 3 Zia Validator
CITIZEN Zuzalu 4 Zeke Governor
STATE Prospera 300
CITIZEN Prospera 5 Pax Builder
CITIZEN Prospera 6 Pia Researcher
ALLIANCE_NEW RedNet
ALLIANCE_ADD RedNet Zuzalu
ALLIANCE_ADD RedNet Prospera
POLY_SUMMARIES
