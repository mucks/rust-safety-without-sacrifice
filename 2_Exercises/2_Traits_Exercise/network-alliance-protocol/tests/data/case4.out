[State] Zuzalu: pop=4 gdp=400
[Alliance] RedNet: members=2 total_pop=6
